src/Type/Definition/FieldArgument.php

```diff
@@ -4,10 +4,13 @@
 
 namespace GraphQL\Type\Definition;
 
+use GraphQL\Error\Error;
 use GraphQL\Error\InvariantViolation;
 use GraphQL\Language\AST\InputValueDefinitionNode;
+use GraphQL\Type\Schema;
 use GraphQL\Utils\Utils;
 use function array_key_exists;
+use function assert;
```

Comment:
Might be unused. Wonder why it was not caught by CS check 🤔

Reply:
Right you are. fdcce6e